guides/graphite-cli/cli-command-reference.md

```diff
@@ -205,11 +219,11 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main or suggests to resubmit a branch whose PR base has changed.
+    * Suppress prompts when Graphite makes suggestions for deleting a branch which is already merged into main or to resubmit a branch whose PR base has changed.
```

Comment:
This part still sounds like Graphite is still making suggestions, but they’re simply being suppressed.

If what we’re saying is Graphite will not make the suggestion in the first place. Then I would suggest changing this to follow the same pattern as the other lines:

“Skip the steps where Graphite checks whether feature branches have been merged into trunk, and suggests resubmitting branches whose PR bases differ locally from remote."

Reply:
So we don't skip the check, we just suppress the prompt and take default action.

Reply:
Ah I see, what is the default action? Is it a user setting?

Reply:
Sometimes. If they supply additional options like `--draft` then we use that, else we choose the conservative option based on the default choices indicated in the `--help` menu.

Reply:
Okay, then maybe something like

"Skip prompting to delete feature branches that have been merged into trunk, and suggesting resubmission of branches whose PR bases differ locally from remote. Instead, Graphite will take the default actions ..."

Followed by explaining what the default action is or how it's set